installation/rpm.adoc

```diff
@@ -10,4 +10,43 @@
 
 toc::[]
 
-== Overview
\ No newline at end of file
+== Overview
+
+You can install OpenShift using RPM packages. Consult your Red Hat account representative for more details on gaining access to the appropriate repositories during the OpenShift Enterprise 3.0 Beta period.
```

Comment:
I am guessing that there will be publicly available RPMs, so I'd probably reword this as a OpenShift general "coming soon"